ranger/data/scope.sh

```diff
@@ -46,6 +46,8 @@ HIGHLIGHT_OPTIONS="--replace-tabs=${HIGHLIGHT_TABWIDTH} --style=${HIGHLIGHT_STYL
 PYGMENTIZE_STYLE=${PYGMENTIZE_STYLE:-autumn}
 OPENSCAD_IMGSIZE=${RNGR_OPENSCAD_IMGSIZE:-1000,1000}
 OPENSCAD_COLORSCHEME=${RNGR_OPENSCAD_COLORSCHEME:-Tomorrow Night}
+SQLITE_TABLE_LIMIT=20  # Display only the top <limit> tables in database, set to 0 for no exhaustive preview (only the sqlite_master table is displayed).
```

Comment:
I think we usually document such ranger specific variables in the man page, if there are non-specific variables that's even better.

Reply:
Some db files are really big, I add these variable for performance reasons, such as `HIGHLIGHT_SIZE_MAX` above.

Reply:
That's ok, but do check whether sqlite3 already has a way of configuring this. The `bat` utility is one example where we had custom variables even though it already respects some.
And either way they should be *documented* in the man page.